.github/workflows/apk-s3-distribute.yml

```diff
@@ -10,6 +10,7 @@ jobs:
   build_ui_components_sample_app:
     name: Build and push to S3
     runs-on: ubuntu-18.04
+    environment: 'publish'
```

Comment:
Shouldn't we set the environment in `ReleaseStart` instead of here and `publish`? `ReleaseStart` is the first script and I think we shouldn't run anything without initial approve  🤔